docs/assets/style.css

```diff
@@ -46,16 +46,92 @@ p a {
     text-decoration: underline;
 }
 h1{
-    font-size: 140% !important;
+    font-size: 145% !important;
     font-weight: bold;
 }
 h2{
-    font-size: 130% !important;
+    font-size: 140% !important;
     font-weight: bold;
 }
 h3{
-    font-size: 115% !important;
+    font-size: 125% !important;
     font-weight: bold;
 }
+.amr-option-group {
+    padding: 24px;
+    width: 100%;
+    background-color: rgb(17, 24, 39);
+
+}
+.side-picture-background {
+    background-color: rgb(17, 24, 39);
+}
+
+.side-image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 30%;
+
+}
+div.ext-box { display: table; height: 100%; width:100%;}
+div.int-box { display: table-cell; vertical-align: middle; }
+.radio-group{
+    width: 100%;
+}
+.form-control{
+    transition: all 1.9s;
+}
+.radio label {
+    margin-top: 12px;
+    padding-top: 24px;
+    padding-bottom: 24px;
+    padding-left: 24px;
+    transition-timing-function: ease-out;
+    transition: all 1s;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    box-shadow: 0 3px 6px rgba(145, 145, 145, 0.16), 0 3px 6px rgba(89, 89, 89, 0.23);
+}
+.bg-troubleshooter{
```

Comment:
added a comment here about the `troubleshooting.png` asset, why not just use the logo with padding instead of forcing an asset like that?

Reply:
It is actually done like that. The asset was just a placeholder for developing it. Forgot to remove it.